samples/wifi/shell/src/wifi_main.c

```diff
@@ -7,6 +7,41 @@
 #include <zephyr/sys/printk.h>
 #include <nrfx_clock.h>
 
+#include <zephyr/kernel.h>
+#include <zephyr/net/ethernet.h>
+#include <zephyr/net/ethernet_mgmt.h>
+
+static void test_change_mac(void)
```

Comment:
This should part of unit tests (https://docs.zephyrproject.org/latest/develop/test/ztest.html) and not in the sample.

Reply:
This was more of an example on how the user needs to configure MAC address.

Reply:
Well, we are trying to configure Nordic OUI mac address and as this is a shell sample, this is what everyone will use, so, I wouldn't recommend putting this in the sample, may be we can have a separate sample to demonstrate MAC address change with a proper Kconfig from the user.